src/client/ducks/repositories/queries/changelog.graphql

```diff
@@ -0,0 +1,15 @@
+query repositoryBranches($name: String!) {
+  repository(name: $name){
+    changelog (limit: 100, offset: 0) {
+      total
+      nodes {
+        id
+        message
+        date
+        status
+        branch
+        author      
```

Comment:
I am not sure this deployed to ono-staging yet, so use your local kallithea instance and pull latest from branch `unity`.